src/actions/index.ts

```diff
@@ -268,24 +284,48 @@ const _refreshOperations: ThunkAction<
 export const refreshOperations: ActionCreator<typeof _refreshOperations> = () =>
   _refreshOperations;
 
-export const applyOperation: (
-  op: APIOperation
-) => ThunkAction<void, AppState, void> = (operation: APIOperation) => async (
-  dispatch,
-  getState
-) => {
-  dispatch({
-    type: OperationsActionType.ADD_OPERATION,
-    operation
-  });
+export const applyOperation: AsyncActionCreator = (
+  operation: ApiOperation
+) => async (dispatch, getState) => {
+  const action: AddOperationsAction = {
+    type: OperationsActionType.ADD_OPERATIONS,
+    operations: [operation]
+  };
+  dispatch(action);
 };
 
-export type MembersAction = SetOperationsAction | AddOperationAction;
-export const refreshMembers: ActionCreator<
-  ThunkAction<void, AppState, void>
-> = () => {
+export type MembersAction = SetOperationsAction | AddOperationsAction;
+export const refreshMembers: AsyncActionCreator = () => {
   return async (dispatch, getState) => {
     // TODO: make this request cached members, not reconstruct from operations
     await _refreshOperations(dispatch, getState, undefined);
   };
 };
+
+export const trustMember: AsyncActionCreator = (uid: Uid) => {
+  return wrapApiCallAction(
+    async (dispatch, getState) => {
+      const authToken = await getAuthToken(getState());
+      if (!authToken) {
+        throw new UnauthenticatedError();
+      }
+
+      const response = await callApi<TrustMemberApiEndpoint>(
+        {
+          endpoint: ApiEndpoint.TRUST_MEMBER,
+          params: { uid },
+          body: undefined
+        },
+        authToken
+      );
+
+      const action: OperationsAction = {
+        type: OperationsActionType.ADD_OPERATIONS,
+        operations: [response]
+      };
+      dispatch(action);
+    },
+    ApiEndpoint.TRUST_MEMBER,
+    uid
```

Comment:
What do you think about moving the `endpoint` and `identifier` arguments ahead of `asyncAction`? Having the two args after the action function is a bit hard to read.